src/Hiccup.jl

```diff
@@ -15,9 +20,15 @@ tag{T}(node::Node{T}) = T
 attrs(node::Node) = node.attrs
 children(node::Node) = node.children
 
-Node(tag::Symbol, attrs::Dict = Dict(), content::AbstractVector = c(); kws...) =
-  Node{tag}(isempty(kws) ? attrs : merge(attrs, Dict([kws...])),
-            content)
+isvoid(tag::Symbol) = tag in VOID_ELEMENTS
+isvoid{T}(node::Node{T}) = isvoid(T)
```

Comment:
Just as a small style point, I think this would be cleaner as `isvoid(node::Node) = isvoid(tag(node))`